> @rultor merge it please

@amihaiemil @lukacupic Oops, I failed. You can see the full log [here](https://www.rultor.com/t/23710-728173492) (spent 2min)

```
[ERROR] /home/r/repo/src/main/java/com/selfxdsd/todocli/JsonTodo.java:19: Comment has incorrect indentation level 16, expected is 0, indentation should be the same level as line 20. [CommentsIndentation]
[ERROR] /home/r/repo/src/main/java/com/selfxdsd/todocli/JsonTodo.java:20: Comment has incorrect indentation level 0, expected is 16, indentation should be the same level as line 21. [CommentsIndentation]
[ERROR] /home/r/repo/src/main/java/com/selfxdsd/todocli/AbstractJsonObject.java:67:5: Method 'getJsonArray' is not designed for extension - needs to be abstract, final or empty. [DesignForExtension]
[ERROR] /home/r/repo/src/main/java/com/selfxdsd/todocli/AbstractJsonObject.java:72:5: Method 'getJsonObject' is not designed for extension - needs to be abstract, final or empty. [DesignForExtension]
[ERROR] /home/r/repo/src/main/java/com/selfxdsd/todocli/AbstractJsonObject.java:77:5: Method 'getJsonNumber' is not designed for extension - needs to be abstract, final or empty. [DesignForExtension]
[ERROR] /home/r/repo/src/main/java/com/selfxdsd/todocli/AbstractJsonObject.java:82:5: Method 'getJsonString' is not designed for extension - needs to be abstract, final or empty. [DesignForExtension]
[ERROR] /home/r/repo/src/main/java/com/selfxdsd/todocli/AbstractJsonObject.java:87:5: Method 'getString' is not designed for extension - needs to be abstract, final or empty. [DesignForExtension]
[ERROR] /home/r/repo/src/main/java/com/selfxdsd/todocli/AbstractJsonObject.java:92:5: Method 'getString' is not designed for extension - needs to be abstract, final or empty. [DesignForExtension]
[ERROR] /home/r/repo/src/main/java/com/selfxdsd/todocli/AbstractJsonObject.java:97:5: Method 'getInt' is not designed for extension - needs to be abstract, final or empty. [DesignForExtension]
[ERROR] /home/r/repo/src/main/java/com/selfxdsd/todocli/AbstractJsonObject.java:102:5: Method 'getInt' is not designed for extension - needs to be abstract, final or empty. [DesignForExtension]
[ERROR] /home/r/repo/src/main/java/com/selfxdsd/todocli/AbstractJsonObject.java:107:5: Method 'getBoolean' is not designed for extension - needs to be abstract, final or empty. [DesignForExtension]
[ERROR] /home/r/repo/src/main/java/com/selfxdsd/todocli/AbstractJsonObject.java:112:5: Method 'getBoolean' is not designed for extension - needs to be abstract, final or empty. [DesignForExtension]
[ERROR] /home/r/repo/src/main/java/com/selfxdsd/todocli/AbstractJsonObject.java:117:5: Method 'isNull' is not designed for extension - needs to be abstract, final or empty. [DesignForExtension]
[ERROR] /home/r/repo/src/main/java/com/selfxdsd/todocli/AbstractJsonObject.java:122:5: Method 'getValue' is not designed for extension - needs to be abstract, final or empty. [DesignForExtension]
[ERROR] /home/r/repo/src/main/java/com/selfxdsd/todocli/AbstractJsonObject.java:127:5: Method 'getValueType' is not designed for extension - needs to be abstract, final or empty. [DesignForExtension]
[ERROR] /home/r/repo/src/main/java/com/selfxdsd/todocli/AbstractJsonObject.java:132:5: Method 'asJsonObject' is not designed for extension - needs to be abstract, final or empty. [DesignForExtension]
[ERROR] /home/r/repo/src/main/java/com/selfxdsd/todocli/AbstractJsonObject.java:137:5: Method 'asJsonArray' is not designed for extension - needs to be abstract, final or empty. [DesignForExtension]
[ERROR] /home/r/repo/src/main/java/com/selfxdsd/todocli/AbstractJsonObject.java:142:5: Method 'toString' is not designed for extension - needs to be abstract, final or empty. [DesignForExtension]
[ERROR] /home/r/repo/src/main/java/com/selfxdsd/todocli/AbstractJsonObject.java:147:5: Method 'size' is not designed for extension - needs to be abstract, final or empty. [DesignForExtension]
[ERROR] /home/r/repo/src/main/java/com/selfxdsd/todocli/AbstractJsonObject.java:152:5: Method 'isEmpty' is not designed for extension - needs to be abstract, final or empty. [DesignForExtension]
[ERROR] /home/r/repo/src/main/java/com/selfxdsd/todocli/AbstractJsonObject.java:157:5: Method 'containsKey' is not designed for extension - needs to be abstract, final or empty. [DesignForExtension]
[ERROR] /home/r/repo/src/main/java/com/selfxdsd/todocli/AbstractJsonObject.java:162:5: Method 'containsValue' is not designed for extension - needs to be abstract, final or empty. [DesignForExtension]
[ERROR] /home/r/repo/src/main/java/com/selfxdsd/todocli/AbstractJsonObject.java:167:5: Method 'get' is not designed for extension - needs to be abstract, final or empty. [DesignForExtension]
[ERROR] /home/r/repo/src/main/java/com/selfxdsd/todocli/AbstractJsonObject.java:172:5: Method 'put' is not designed for extension - needs to be abstract, final or empty. [DesignForExtension]
[ERROR] /home/r/repo/src/main/java/com/selfxdsd/todocli/AbstractJsonObject.java:177:5: Method 'remove' is not designed for extension - needs to be abstract, final or empty. [DesignForExtension]
[ERROR] /home/r/repo/src/main/java/com/selfxdsd/todocli/AbstractJsonObject.java:182:5: Method 'putAll' is not designed for extension - needs to be abstract, final or empty. [DesignForExtension]
[ERROR] /home/r/repo/src/main/java/com/selfxdsd/todocli/AbstractJsonObject.java:187:5: Method 'clear' is not designed for extension - needs to be abstract, final or empty. [DesignForExtension]
[ERROR] /home/r/repo/src/main/java/com/selfxdsd/todocli/AbstractJsonObject.java:192:5: Method 'keySet' is not designed for extension - needs to be abstract, final or empty. [DesignForExtension]
[ERROR] /home/r/repo/src/main/java/com/selfxdsd/todocli/AbstractJsonObject.java:197:5: Method 'values' is not designed for extension - needs to be abstract, final or empty. [DesignForExtension]
[ERROR] /home/r/repo/src/main/java/com/selfxdsd/todocli/AbstractJsonObject.java:202:5: Method 'entrySet' is not designed for extension - needs to be abstract, final or empty. [DesignForExtension]
[ERROR] /home/r/repo/src/main/java/com/selfxdsd/todocli/AbstractJsonObject.java:207:5: Method 'equals' is not designed for extension - needs to be abstract, final or empty. [DesignForExtension]
[ERROR] /home/r/repo/src/main/java/com/selfxdsd/todocli/AbstractJsonObject.java:212:5: Method 'hashCode' is not designed for extension - needs to be abstract, final or empty. [DesignForExtension]
[ERROR] /home/r/repo/src/main/java/com/selfxdsd/todocli/AbstractJsonObject.java:217:5: Method 'getOrDefault' is not designed for extension - needs to be abstract, final or empty. [DesignForExtension]
[ERROR] /home/r/repo/src/main/java/com/selfxdsd/todocli/AbstractJsonObject.java:222:5: Method 'forEach' is not designed for extension - needs to be abstract, final or empty. [DesignForExtension]
[ERROR] /home/r/repo/src/main/java/com/selfxdsd/todocli/AbstractJsonObject.java:227:5: Method 'replaceAll' is not designed for extension - needs to be abstract, final or empty. [DesignForExtension]
[ERROR] /home/r/repo/src/main/java/com/selfxdsd/todocli/AbstractJsonObject.java:232:5: Method 'putIfAbsent' is not designed for extension - needs to be abstract, final or empty. [DesignForExtension]
[ERROR] /home/r/repo/src/main/java/com/selfxdsd/todocli/AbstractJsonObject.java:237:5: Method 'remove' is not designed for extension - needs to be abstract, final or empty. [DesignForExtension]
[ERROR] /home/r/repo/src/main/java/com/selfxdsd/todocli/AbstractJsonObject.java:242:5: Method 'replace' is not designed for extension - needs to be abstract, final or empty. [DesignForExtension]
[ERROR] /home/r/repo/src/main/java/com/selfxdsd/todocli/AbstractJsonObject.java:247:5: Method 'replace' is not designed for extension - needs to be abstract, final or empty. [DesignForExtension]
[ERROR] /home/r/repo/src/main/java/com/selfxdsd/todocli/AbstractJsonObject.java:252:5: Method 'computeIfAbsent' is not designed for extension - needs to be abstract, final or empty. [DesignForExtension]
[ERROR] /home/r/repo/src/main/java/com/selfxdsd/todocli/AbstractJsonObject.java:257:5: Method 'computeIfPresent' is not designed for extension - needs to be abstract, final or empty. [DesignForExtension]
[ERROR] /home/r/repo/src/main/java/com/selfxdsd/todocli/AbstractJsonObject.java:262:5: Method 'compute' is not designed for extension - needs to be abstract, final or empty. [DesignForExtension]
[ERROR] /home/r/repo/src/main/java/com/selfxdsd/todocli/AbstractJsonObject.java:267:5: Method 'merge' is not designed for extension - needs to be abstract, final or empty. [DesignForExtension]
[ERROR] /home/r/repo/src/test/java/com/selfxdsd/todocli/JsonTodoTest.java:19:5: Method 'whenCallingCtorOfJsonTodoExpectJsonObject' is not designed for extension - needs to be abstract, final or empty. [DesignForExtension]
Audit done.
[\u001b[1;34mINFO\u001b[m] \u001b[1m------------------------------------------------------------------------\u001b[m
[\u001b[1;34mINFO\u001b[m] \u001b[1;31mBUILD FAILURE\u001b[m
[\u001b[1;34mINFO\u001b[m] \u001b[1m------------------------------------------------------------------------\u001b[m
[\u001b[1;34mINFO\u001b[m] Total time:  9.847 s
[\u001b[1;34mINFO\u001b[m] Finished at: 2020-11-16T16:33:38Z
[\u001b[1;34mINFO\u001b[m] \u001b[1m------------------------------------------------------------------------\u001b[m
[\u001b[1;31mERROR\u001b[m] Failed to execute goal \u001b[32morg.apache.maven.plugins:maven-checkstyle-plugin:3.0.0:check\u001b[m \u001b[1m(validate)\u001b[m on project \u001b[36mtodo-finder-cli\u001b[m: \u001b[1;31mFailed during checkstyle execution\u001b[m: There are 102 errors reported by Checkstyle 6.18 with checkstyle.xml ruleset. -> \u001b[1m[Help 1]\u001b[m
[\u001b[1;31mERROR\u001b[m] 
[\u001b[1;31mERROR\u001b[m] To see the full stack trace of the errors, re-run Maven with the \u001b[1m-e\u001b[m switch.
[\u001b[1;31mERROR\u001b[m] Re-run Maven using the \u001b[1m-X\u001b[m switch to enable full debug logging.
[\u001b[1;31mERROR\u001b[m] 
[\u001b[1;31mERROR\u001b[m] For more information about the errors and possible solutions, please read the following articles:
[\u001b[1;31mERROR\u001b[m] \u001b[1m[Help 1]\u001b[m http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
container 5c97f161f76c29b26ad77a657a288d162c9a9f83dc09148d8df269dfc06f9922 is dead
Mon Nov 16 17:33:48 CET 2020
```

--- a/src/main/java/com/selfxdsd/todocli/Todo.java
+++ b/src/main/java/com/selfxdsd/todocli/Todo.java
@@ -22,9 +22,6 @@
  */
 package com.selfxdsd.todocli;
 
-// @todo #4:60min  Add interface Todos which should represent an iterable of Todo objects.
-//  Afterwards, add class JsonTodos, which should represent the TODOs as JsonArray.
-
 /**
  * Representation of a TODO or FIXME.
  *
@@ -32,6 +29,8 @@ package com.selfxdsd.todocli;
  * @checkstyle AbbreviationAsWordInName (500 lines)
  * @checkstyle FinalParameters (500 lines)
  * @since 0.0.1
+ * @todo #4:60min  Add interface Todos which should represent an iterable of Todo objects.
+ *  Afterwards, add class JsonTodos, which should represent the TODOs as JsonArray.
  */
 public final class Todo {
 

